lib/cmd/fhc/target.js

```diff
@@ -137,11 +137,12 @@ function undefinedTargCleanup(cb, tar) {
 }
 
 function pingTarget(targ, callback) {
-  request(targ + "/box/srv/1.1/act/sys/auth/logout", function(err, response, body){
+  fhreq.GET(targ, "/box/srv/1.1/act/sys/auth/logout", i18n._("error pinging target"), function(err, parsed, _, response) {
```

Comment:
Yes, the `fhreq` wrapper will automatically add the `User-Agent` header to the request. Thats why we should always use it in FHC commands.